setup.py

```diff
@@ -1,9 +1,11 @@
-from distutils.core import setup
+from setuptools import setup, find_packages
+PACKAGES = find_packages()
 
 if __name__ == '__main__':
     setup(
     name='BrainNetworksInPython',
     version='0.1dev',
-    packages=['scripts','wrappers'],
+    packages=PACKAGES,
```

Comment:
Great! I'd love to understand this a little better - only if you fancy explaining when we next meet 😸

Reply:
Coolio - and we want to *install* the data because we're shipping it with the package (rather than, for example, just reading it in from the github repo), right?